src/limbo/tools/random_generator.hpp

```diff
@@ -54,6 +54,16 @@ namespace limbo {
         }
 
         /// @ingroup tools
+        /// make a 1-D vector from a double (useful when we need to return vectors)
+        Eigen::VectorXd make_vector(double x)
```

Comment:
Does it make sense to have the `make_vector` function in the `random_generator` file?